### DISCUSSION TITLE
Unable to install oh-my-posh

### DISCUSSION THREAD

**Discussion by aaridoms** [JanDeDobbeleer/oh-my-posh#5946] [Troubleshoot]
I just reinstalled windows and when I went to install oh-my-posh via scoop I get the following error:

![image](https://github.com/user-attachments/assets/0188a8ef-4522-4a9c-a04e-5967ee0693c3)

![image](https://github.com/user-attachments/assets/f6cf83a6-7346-4c98-9d20-f118360d54bc)

**Comment by JanDeDobbeleer** [comment]
@aaridoms there's an issue for this already, I need to look into it but I'm sure it can still be supported.
